### ISSUE TITLE
Accessibility: custom create, config map, workload

### ISSUE THREAD

**Issue by lvuch** [rancher/dashboard#4394]
Master https://github.com/rancher/dashboard/issues/4324

16.
## Page url: [https://a11y.rancher.space/dashboard/c/\_/manager/provisioning.cattle.io.cluster/create?type=custom](https://a11y.rancher.space/dashboard/c/_/manager/provisioning.cattle.io.cluster/create?type=custom)

**Light theme** issues encountered **:**

- ![](RackMultipart20211014-4-19dhorn_html_9cf6231a6f7f9469.png) **image referenced**

**Dark theme** issues encountered **:**

**------**

17.
##

## Shared

- Similar actions use different visual cues (icon in this case). In the top toolbar we have ![](RackMultipart20211014-4-19dhorn_html_72511a35aceb0639.png) **image referenced** for download while in the button to download a YAML file we have ![](RackMultipart20211014-4-19dhorn_html_6166365082de0ae.png) **image referenced**. This is against the criteria [3.2.4 &quot;consistent identification&quot;](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

**------**

18.
##

## Shared

- On each of the tbody there is a title that describes the content of the tbody. This title is currently embedded in a \&lt;td\&gt; but it should be changed to a \&lt;th\&gt;. [More on this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody#multiple_bodies).
 ![](RackMultipart20211014-4-19dhorn_html_546de92bfe6be51f.png) **image referenced**

**Comment by catherineluse** [comment]
I'm closing this issue because the remaining work to do for accessibility is being tracked in this ticket: https://github.com/rancher/dashboard/issues/4324
